### PR TITLE
Feat/customizable formats

### PR DESCRIPTION
**Pull request by namelot**

### CODE REVIEW

File: thumbor.js [hunk 1]
```diff
@@ -52,24 +52,22 @@ class CpmThumbor{
...
+        const res = {};
 
-        return {
-            "xs": this.buildUrl(mediaUrl, parametersXs),
-            "md": this.buildUrl(mediaUrl, parametersMd),
-            "lg": this.buildUrl(mediaUrl, parametersLg),
-            "original": this.buildUrl(mediaUrl, {})
-        };
+        formats.forEach(format => {
+            res[format] = this.buildUrl(mediaUrl, Object.assign({ ... rewriteParameters }, this.getFormat(format)))
```
Comment (Eygle) [inline thumbor.js]:
```suggestion
            res[format] = this.buildUrl(mediaUrl, { ... rewriteParameters, ...this.getFormat(format) })
```

File: thumbor.js [hunk 2]
```diff
@@ -52,32 +50,34 @@ class CpmThumbor{
...
+        return res;
     }
 
     buildUrl (mediaUrl, rewriteParameters) {
         const url = new Url(mediaUrl);
 
         let transform = {};
         if (rewriteParameters.transform) {
-            transform = pick(rewriteParameters.transform, AUTHORIZED_METAS);
+            for (let [key, value] of Object.entries(rewriteParameters.transform)) {
```
Comment (wako057) [inline thumbor.js]: const?

Comment (namelot) [reply thumbor.js]: Fix => 1c7c6299988b9609f75d4596ae2c42ce75076b18

File: thumbor.js (hunk 1 above)
Comment (namelot) [reply thumbor.js]: Fix 72dd5307a48a9cfbb5669f62d57e15f5a882e836

**Review by Eygle — APPROVED** [review]

**Review by wako057 — APPROVED** [review]

**Pull request merged**